packages/create-app/src/lib/tasks.ts

```diff
@@ -236,3 +236,34 @@ export async function moveAppTask(
       });
   });
 }
+
+/**
+ * Initializes a git repository in the destination folder
+ *
+ * @param dir - source path to initialize git repository in
+ * @param context - template parameters
+ * @throws if `exec` fails
+ */
+export async function initGitRepository(dir: string, context: any) {
+  const runCmd = async (cmd: string) => {
+    let cmdResponse = { stdout: '', stderr: '' };
```

Comment:
No need for the separate declaration, the `Task.forItem` method will forward whatever you return from it